test/e2e/e2e.go

```diff
@@ -1659,6 +1659,58 @@ var _ = ginkgo.Describe("e2e ingress gateway traffic validation", func() {
 	})
 })
 
+// This test perform a longevity test, where it spawns multiple container
+// and wait for them to finish executing sleep. It does this exercise for
+// 20 minutes and stops. Each container starts, sleeps for 10 seconds and
+// goes to success or failure state.
```

Comment:
nice simple longevity test :)

I think a cool extension to this (maybe I can take it up at some point) would be to add some network activity from
each pod that comes up to "somewhere" (another pod, service, node, etc) and then parse the pod log to make
sure that was successful. Could catch some corner cases or races. just thinking out loud.

Reply:
@jluhrsen We can run iperf client/server in these pods and log/evaluate the result. That would be good improvement, but i am not sure if github infra will give us enough resources to run the test with some level of stability.

The ideal behind this test was to give a bit of sanity check for upstream patches. It creates around 600-700 pods in this test, so atleast it will be helpful to surface any low hanging bugs. But by no means this test is enough. We should have a proper longevity test with more chaos tests running to really check the stability of the code.

Reply:
agreed. I'll add a story to my teams back-log so I can pick it up at some point.